### PR TITLE
add awaiting review button

### PR DESCRIPTION
**Pull request by MelizzaP**
added the awaiting review button to cms homepage

test link: http://bulbs-cms-bump.test.clickhole.com/cms

### CODE REVIEW

**Comment by MelizzaP** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;lBLosj2q&#x2F;551-webtech-clickhole-cms-bug-report"> [WEBTECH] Clickhole CMS Bug Report</a>

**Comment by MelizzaP** [comment]
@spra85 @collin

**Comment by collin** [comment]
The case of `'draft'` doesn't match `'Waiting for Editor'`

Is that intended?

File: app/components/status-filter/status-filter-directive.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ angular.module('statusFilter.directive', [
   .provider('StatusFilterOptions', function (moment) {
     var _statuses = [
       {label: 'Draft', key: 'status', value: 'draft'},
+      {label: 'Awaiting Review', key: 'status', value: 'Waiting for Editor'},
```
Comment (collin) [inline app/components/status-filter/status-filter-directive.js]: Here, `value` case doesn't match.

Comment (MelizzaP) [inline app/components/status-filter/status-filter-directive.js]: good catch, looks like it should be `Draft`

Comment (MelizzaP) [inline app/components/status-filter/status-filter-directive.js]: edit: looks like they are both case sensitive and opposite... not sure how that happened

Comment (collin) [inline app/components/status-filter/status-filter-directive.js]: No porbs. Just wanted to make sure.

**Comment by collin** [comment]
👍

**Comment by spra85** [comment]
Is this the new default?  I feel like we should be making sure this doesn't change the filters for The A.V. Club because I feel like they're different

**Comment by MelizzaP** [comment]
avclub is on an older version of bulbs-cms until kos gets back https://github.com/theonion/avclub/pull/522

**Comment by MelizzaP** [comment]
looks like avclub won't be affected its a major version behind https://github.com/theonion/avclub/blob/master/bower.json#L16

**Pull request merged**